framework/CodeInterfaces/RAVEN/RAVENInterface.py

```diff
@@ -38,13 +38,13 @@ def __init__(self):
     self.printTag  = 'RAVEN INTERFACE'
     self.outputPrefix = 'out~'
     self.outStreamsNamesAndType = {} # Outstreams names and type {'outStreamName':[DataObjectName,DataObjectType]}
-    # pointer to the module that contains the function to modify and convert the sampled vars (optional)
+    # path to the module that contains the function to modify and convert the sampled vars (optional)
     # 2 methods are going to be inquired (if present and needed):
     # - convertNotScalarSampledVariables
     # - manipulateScalarSampledVariables
-    self.extModForVarsManipulation = None
-    # 'noscalar' = True if convertNotScalarSampledVariables exists in self.extModForVarsManipulation module
-    # 'scalar'   = True if manipulateScalarSampledVariables exists in self.extModForVarsManipulation module
+    self.extModForVarsManipulationPath = None
+    # 'noscalar' = True if convertNotScalarSampledVariables exists in extModForVarsManipulation module
+    # 'scalar'   = True if manipulateScalarSampledVariables exists in extModForVarsManipulation module
```

Comment:
I changed it in order to avoid the pickling when the input creation is done in parallel. 
In this way, we import the function just in case it is needed in the createNewInput

Reply:
OK